internal/db/schema/migrations/oss/postgres/29/01_cancel_session_null_fkey.up.sql

```diff
@@ -0,0 +1,37 @@
+begin;
+
+drop function cancel_session(in sessionId text);
+-- Updates cancel_session() from 0/50_session to check if a session is either terminated or canceling
+-- Sessions can progress directly to terminated without going through the canceling state
+-- cancel_session will insert a cancel state for the session, if there's isn't
+-- a canceled or terminated state already.  It's used by cancel_session_with_null_fk.
+create function
+  cancel_session(in sessionId text) returns void
```

Comment:
Can we add unit tests for this?

Reply:
Sorry, I should have been more clear. I was thinking the unit tests would be added to our sql test suite since this is a sql function. Would it be possible to add unit tests there?